src/components/Anchor/index.js

```diff
@@ -0,0 +1,41 @@
+// https://govuk-elements.herokuapp.com/typography/
+
+import React from "react";
+import PropTypes from "prop-types";
+import glamorous from "glamorous";
+
+import { BLUE, PURPLE, YELLOW } from "govuk-colours";
+
+const AnchorInner = glamorous.a({
```

Comment:
Many applications will want to use a React Router `<Link />` component for anchors, but have it styled by glamorous.

We don't want to introduce a reliance on React Router, but this component should be configurable to support rendering/extending Link.

Could you look if there are similar styled-components/glamorous examples that do this? Or read the react router docs for Link and see if you can find any clear ways to do this?

Even better would be good to have a 'usage with react router' example in storybook.